build/rpm/services/milvus-etcd.service

```diff
@@ -27,4 +27,4 @@ StandardError=inherit
 Restart=always
 
 # Start main service
-ExecStart=/usr/bin/milvus-etcd --data-dir /etc/milvus/etcd-data
```

Comment:
@LoveEachDay: What is the correct value for `--data-dir`? 

In the `master` branch it is `--data-dir /etc/milvus/etcd-data`.
But when I `apt-get install milvus==2.0.2-1` the content of `/etc/systemd/system/milvus-etcd.service` is different: ` --data-dir /var/lib/milvus/etcd-data`. 

![image](https://user-images.githubusercontent.com/17709711/165825279-3d72f95d-0a37-4be1-80d5-26bbdd279055.png)


`/var/lib/milvus/etcd-data/` exists and holds files, `/etc/milvus/etcd-data` does not exist.

Reply:
We change the default data directory for milvus 2.0.2.
/cc @Bennu-Li

Reply:
For ubuntu the default data directory for etcd is `/var/lib/milvus/ectd-data`, and for centos related the default directory for etcd is `/etc/milvus/etcd-data` .

And for now we don't put the service config for ubuntu into milvus repo.